### PR TITLE
Mount the SSH keys as secrets in all workspace containers: OpenShift Env

### PR DESCRIPTION
**Pull request by vparfonov**
Signed-off-by: Vitalii Parfonov <vparfono@redhat.com>

### What does this PR do?
Mount the SSH keys as secrets in all workspace containers: OpenShift Env

### What issues does this PR fix or reference?
#13520

### CODE REVIEW

**Comment by vparfonov** [comment]
ci-test

**Review by amisevsk — APPROVED** [review]

**Comment by musienko-maxim** [comment]
Can one of the admins verify this PR?

File: infrastructures/openshift/src/main/java/org/eclipse/che/workspace/infrastructure/openshift/OpenShiftEnvironmentProvisioner.java [hunk 1]
```diff
@@ -133,6 +137,7 @@ public void provision(OpenShiftEnvironment osEnv, RuntimeIdentity identity)
     proxySettingsProvisioner.provision(osEnv, identity);
     serviceAccountProvisioner.provision(osEnv, identity);
     certificateProvisioner.provision(osEnv, identity);
+    vcsSshKeysProvisioner.provision(osEnv, identity);
```
Comment (sleshchenko) [inline infrastructures/openshift/src/main/java/org/eclipse/che/workspace/infrastructure/openshift/OpenShiftEnvironmentProvisioner.java:140]: Please add the corresponding check in tests.

**Review by sleshchenko — APPROVED** [review]
LGTM

**Review by rhopp — APPROVED** [review]

File: infrastructures/openshift/src/main/java/org/eclipse/che/workspace/infrastructure/openshift/OpenShiftEnvironmentProvisioner.java (hunk 1 above)
Comment (vparfonov) [reply infrastructures/openshift/src/main/java/org/eclipse/che/workspace/infrastructure/openshift/OpenShiftEnvironmentProvisioner.java:140]: done

**Pull request merged**